deploy.sh

```diff
@@ -58,6 +58,21 @@ docker run --rm -v $PWD:${MARTHA_PATH} \
 MARTHA_IMAGE=quay.io/broadinstitute/martha:${GIT_BRANCH}
 
 # Overriding ENTRYPOINT has some subtleties: https://medium.com/@oprearocks/how-to-properly-override-the-entrypoint-using-docker-run-2e081e5feb9d
+
+# > Note: As of January 15, 2020, HTTP functions require authentication by default.
+#
+# via: https://cloud.google.com/functions/docs/securing/managing-access-iam#allowing_unauthenticated_function_invocation
+#
+# For every new cloud functions deployed, across every environment, someone with owner access will need to run:
+#   ```
+#   gcloud beta functions \
+#     add-iam-policy-binding {FUNCTION_NAME} \
+#     --member=allUsers --role=roles/cloudfunctions.invoker \
+#     --project=broad-dsde-{ENVIRONMENT}
+#   ```
+# After an owner fixes the perms, the function contents may be overwritten by any editor, including the GCF-SA
+# `secret/dsde/martha/${ENVIRONMENT}/deploy-account.json`, and will stay public.
```

Comment:
nit: I'm assuming GCF stands for Google Cloud Function service account, but it would be nice to spell it out

Reply:
can you provide more details about how the owner would "[fix] the perms"?